From 537bd1c4cb75138ef5ada257eb4c92893efd5652 Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Tue, 13 Aug 2013 17:45:54 -0700
Subject: don't use 'eng' tags, fix header includes

Dbus was removed from AOSP; don't try to build/install it
for all projects. The header mapping was also removed, so
don't try to look it up.

Bug: ADEV-107

Change-Id: I768c8d6c0f6ad7e0a83da27a3089066ffb28fe50
Signed-off-by: Andrew Boie <andrew.p.boie@intel.com>
---
 bus/Android.mk   |    4 ++--
 dbus/Android.mk  |    2 +-
 tools/Android.mk |   12 ++++++------
 3 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/bus/Android.mk b/bus/Android.mk
index 4d5229d..7d11b70 100644
--- a/bus/Android.mk
+++ b/bus/Android.mk
@@ -2,8 +2,8 @@ LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 
 LOCAL_C_INCLUDES:= \
-	$(call include-path-for, dbus) \
-	$(call include-path-for, dbus)/dbus \
+	external/dbus \
+	external/dbus/dbus \
 	external/expat/lib/
 
 LOCAL_CFLAGS:=-O3
diff --git a/dbus/Android.mk b/dbus/Android.mk
index ccd138a..677c913 100644
--- a/dbus/Android.mk
+++ b/dbus/Android.mk
@@ -58,7 +58,7 @@ dbus-watch.c \
 sd-daemon.c \
 
 LOCAL_C_INCLUDES+= \
-	$(call include-path-for, dbus)
+	external/dbus
 
 LOCAL_MODULE:=libdbus
 
diff --git a/tools/Android.mk b/tools/Android.mk
index f6aae50..3fca877 100644
--- a/tools/Android.mk
+++ b/tools/Android.mk
@@ -5,13 +5,13 @@ LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES := dbus-print-message.c
-LOCAL_C_INCLUDES += $(call include-path-for, dbus)
+LOCAL_C_INCLUDES += external/dbus
 LOCAL_SHARED_LIBRARIES += libdbus
 LOCAL_CFLAGS += \
 	-DDBUS_COMPILATION \
 	-DDBUS_MACHINE_UUID_FILE=\"/etc/machine-id\"
 LOCAL_MODULE_PATH := $(TARGET_OUT_OPTIONAL_EXECUTABLES)
-LOCAL_MODULE_TAGS := eng
+LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := libdbus-tools-common
 include $(BUILD_STATIC_LIBRARY)
 
@@ -20,14 +20,14 @@ include $(BUILD_STATIC_LIBRARY)
 include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES := dbus-monitor.c
-LOCAL_C_INCLUDES += $(call include-path-for, dbus)
+LOCAL_C_INCLUDES += external/dbus
 LOCAL_SHARED_LIBRARIES += libdbus
 LOCAL_STATIC_LIBRARIES += libdbus-tools-common
 LOCAL_CFLAGS += \
 	-DDBUS_COMPILATION \
 	-DDBUS_MACHINE_UUID_FILE=\"/etc/machine-id\"
 LOCAL_MODULE_PATH := $(TARGET_OUT_OPTIONAL_EXECUTABLES)
-LOCAL_MODULE_TAGS := eng
+LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := dbus-monitor
 include $(BUILD_EXECUTABLE)
 
@@ -36,13 +36,13 @@ include $(BUILD_EXECUTABLE)
 include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES := dbus-send.c
-LOCAL_C_INCLUDES += $(call include-path-for, dbus)
+LOCAL_C_INCLUDES += external/dbus
 LOCAL_SHARED_LIBRARIES += libdbus
 LOCAL_STATIC_LIBRARIES += libdbus-tools-common
 LOCAL_CFLAGS += \
 	-DDBUS_COMPILATION \
 	-DDBUS_MACHINE_UUID_FILE=\"/etc/machine-id\"
 LOCAL_MODULE_PATH := $(TARGET_OUT_OPTIONAL_EXECUTABLES)
-LOCAL_MODULE_TAGS := eng
+LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE := dbus-send
 include $(BUILD_EXECUTABLE)
-- 
1.7.4.1

